air-quality-ui/system_tests/ui/summary_page.spec.ts

```diff
@@ -1,75 +1,807 @@
+        pm10: { aqi_level: 1, value: 10.313057730041718 },
+        so2: { aqi_level: 1, value: 0.465807542763243 },
+      }),
+    ]
+
+    const mockedMeasurementSummaryResponse = [
+      createMeasurementSummaryAPIResponseData({
+        measurement_base_time: '2024-06-19T09:00:00Z',
+        location_name: 'Kampala',
+        overall_aqi_level: { mean: 2 },
+        no2: { mean: { aqi_level: 1, value: 0.3812829140487199 } },
+        o3: { mean: { aqi_level: 2, value: 72.9086035913633 } },
+        pm2_5: { mean: { aqi_level: 6, value: 76 } },
+        pm10: { mean: { aqi_level: 2, value: 26.087666551144732 } },
+        so2: { mean: { aqi_level: 1, value: 0.6314619719025142 } },
+      }),
+      createMeasurementSummaryAPIResponseData({
+        measurement_base_time: '2024-06-19T12:00:00Z',
+        location_name: 'Abu Dhabi',
+        overall_aqi_level: { mean: 5 },
+        no2: { mean: { aqi_level: 1, value: 5.871611751344455 } },
+        o3: { mean: { aqi_level: 4, value: 213.04088459925424 } },
+        pm2_5: { mean: { aqi_level: 5, value: 52.75 } },
+        pm10: { mean: { aqi_level: 4, value: 58.25755291558235 } },
+        so2: { mean: { aqi_level: 1, value: 8.497931484924965 } },
+      }),
+      createMeasurementSummaryAPIResponseData({
+        measurement_base_time: '2024-06-19T12:00:00Z',
+        location_name: 'Zurich',
+        overall_aqi_level: { mean: 2 },
+        no2: { mean: { aqi_level: 1, value: 1.2220194497781245 } },
+        o3: { mean: { aqi_level: 3, value: 110.29793453644987 } },
+        pm2_5: { mean: { aqi_level: 2, value: 15.764618078867594 } },
+        pm10: { mean: { aqi_level: 2, value: 31.71802172436572 } },
+        so2: { mean: { aqi_level: 1, value: 1.3459434727665889 } },
+      }),
+    ]
+    await vairifySummaryPage.setupPageWithMockData(
+      mockedForecastResponse,
+      mockedMeasurementSummaryResponse,
+    )
+  })
+  test.describe('Table Structure', () => {
+    test('Verify that Headers exist and Innertext matches', async ({
+      vairifySummaryPage,
+    }) => {
+      await vairifySummaryPage.getColumnHeaderAndText('AQI Level', 'AQI Level')
+      await vairifySummaryPage.getColumnHeaderAndText(
+        'PM 2.5 (µg/m³)',
+        'PM 2.5 (µg/m³)',
+      )
+      await vairifySummaryPage.getColumnHeaderAndText(
+        'PM 10 (µg/m³)',
+        'PM 10 (µg/m³)',
+      )
+      await vairifySummaryPage.scrollToRightmostPosition()
+      await vairifySummaryPage.page.waitForTimeout(1000)
+
+      await vairifySummaryPage.getColumnHeaderAndText(
+        'Nitrogen Dioxide (µg/m³)',
+        'Nitrogen Dioxide (µg/m³)',
+      )
+      await vairifySummaryPage.getColumnHeaderAndText(
+        'Ozone (µg/m³)',
+        'Ozone (µg/m³)',
+      )
+      await vairifySummaryPage.getColumnHeaderAndText(
+        'Sulphur Dioxide (µg/m³)',
+        'Sulphur Dioxide (µg/m³)',
+      )
+    })
+
+    test('Verify numbers in cells have no more than 1 decimal place ', async ({
```

Comment:
I think this is basically covered by the test that checks the cell contents

Reply:
This test was already in here, I just moved where it is structured, I can remove it though if you think it isn't neccessary